flixel/input/gamepad/OUYAButtonID.hx

```diff
@@ -16,8 +16,8 @@ class OUYAButtonID
 	public static inline var A:Int = 1;
 	public static inline var LB:Int = 6;
 	public static inline var RB:Int = 7;
-	public static inline var LEFT_ANALOG:Int = 10;
-	public static inline var RIGHT_ANALOG:Int = 11;
+	public static inline var LEFT_STICK_BTN:Int = 10;
```

Comment:
CLICK is probably our best bet. They're only buttons in the abstract electronic switch sense, not in terms of physical affordance.